scripts/check-changelog.py

```diff
@@ -29,6 +29,16 @@
 sys.path.append(os.path.dirname(__file__))  # noqa
 
 
+def version_string(nums):
+    print(nums)
+    return '{}.{}.{}'.format(*nums)
```

Comment:
I don't care massively, but would:

```python
'.'.join([str(s) for s in nums])
```

be preferable as it’s more generic? Not that I anticipate us having more than four-part version numbers any time soon, admittedly.